lib/std/spu/interpreter.zig

```diff
@@ -0,0 +1,232 @@
+const std = @import("std");
+usingnamespace @import("defines.zig");
+
+const Bus = struct {
+    ROM: []u8,
+    RAM0: []u8,
+    RAM1: []u8,
```

Comment:
Is there a reason to separate RAM and ROM into seperate parts? It's not really necessary for testing the compiler backend, it's a hardware detail for Ashet, nothing that should matter for a generic backend

Reply:
Good point, I'll probably just give it a full address space of RAM instead or something like that.